src/algorithms/geometric.js

```diff
@@ -4,6 +4,6 @@ export default {
   },
 
   getValue(x, min, max) {
-    return (Math.floor(((x / 100) ** 2) * (max - min)) + min);
```

Comment:
I'm not sure I understand the use case; could you provide tests that would fail without this change?